tests/test_update_check.py

```
@@ -0,0 +1,112 @@
+        with patch(
+            "biblebot.update_check.check_for_updates",
+            side_effect=mock_check_for_updates,
+        ):
+            # Should not raise any exceptions
+            await perform_startup_update_check()
+
+    def test_print_startup_banner(self, caplog):
+        """Test startup banner prints version information."""
+        with caplog.at_level(logging.INFO, logger=LOGGER_NAME):
+            print_startup_banner()
+            assert "Starting BibleBot version" in caplog.text
```

Ruff (0.12.2)

Use of `assert` detected

(S101)